api/src/main/java/net/md_5/bungee/api/event/ProxyInitializeEvent.java

```diff
@@ -0,0 +1,21 @@
+package net.md_5.bungee.api.event;
+
+import java.util.Map;
+import net.md_5.bungee.api.config.ListenerInfo;
+import net.md_5.bungee.api.plugin.Event;
+
+public class ProxyInitializeEvent extends Event
```

Comment:
Also it would be good to "lombokify" this class. Remove the constuctor and getter and instead add `@Data` annotation of lombok.
Javadoc for the getter goes to the field instead.